src/main/java/com/artipie/SliceFromConfig.java

```diff
@@ -243,10 +245,11 @@ static Slice build(final Settings settings, final Authentication auth,
                 final String host = cfg.settings()
                     .orElseThrow(() -> new IllegalStateException("Repo settings not found"))
                     .string("host");
-                slice = new DockerSlice(
-                    cfg.path(),
-                    new ProxyDocker(new ClientSlice(SliceFromConfig.HTTP, host))
-                );
+                final Docker proxy = new ProxyDocker(new ClientSlice(SliceFromConfig.HTTP, host));
+                final Docker docker = cfg.storageOpt()
```

Comment:
@olegmoz can we add a test for this?

Reply:
@Vatavuk I doubt that we can test this class in any meaningful way without significant refactoring. We may check that it returns `DockerSlice`, but cannot check what's inside in unit test

Reply:
@olegmoz what about an IT test?

Reply:
@Vatavuk that would be possible, but there no any IT tests in the project yet